homebox/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: homebox_server_1
```

Comment:
For single container apps we have been naming the container `web` to standardize things.

```suggestion
      APP_HOST: homebox_web_1
```

Reply:
Thanks for bringing this to my attention, been longtime since I made an app suggestion ^^